### ISSUE TITLE
How to judge the quality of training through the loss curves？

### ISSUE THREAD

**Issue by TremblingV5** [matterport/Mask_RCNN#2032]
As we know, Mask_RCNN have some metric value, such as val_loss, val_rpn_class_loss. 
Then, how to judge the quality of training through the loss curves? For example, I want to judge whether it's overfitting.

**Comment by suchiz** [comment]
Hi there, try to smooth your curve accordingly to your number of epoch (large number of epoch, high smoothing factor). Then, if the curve is decreasing, then it's fine even if it is slow. But if the curve is increasing again, then you've reached the overfitting point.

PS: you may have to look on the val_loss, not the training loss. Also, if the training loss is low but the val loss is high, you're probably overfitted.
But i'm pretty sure you can find something more detailed and explained on the internet. That is a common thing of machine learning :)

**Comment by pgirikishore** [comment]
Thank you @suchiz

My training loss is 0.44 and my val loss is 0.63. Is the model overfitting?
